### DISCUSSION TITLE
How to set up a project with tailwindcss, node js, express, and ejs?

### DISCUSSION THREAD

**Discussion by IAmPissed** [tailwindlabs/tailwindcss#9472] [Help]
I'm trying to build a project that utilizes **Node JS**, **Express**, and **ejs**, but I'd like to use **tailwindcss** for styling, how can I do that?
also is there any way to get **auto-reload** every time the styles change, like when you run the server with _nodemon_?

**Comment by wongjn** [comment]
> I'd like to use **tailwindcss** for styling, how can I do that?

You could use the [Tailwind CLI](https://tailwindcss.com/docs/installation). You will need to adjust the `content` file globs to sufficiently cover your templates and other code that uses Tailwind class names.

> is there any way to get auto-reload every time the styles change, like when you run the server with nodemon?

You could try [following this article](https://dev.to/cassiolacerda/automatically-refresh-the-browser-on-node-express-server-changes-x1f680-1k0o), this was found as the first result for an internet search for "express auto reload middleware". If this particular article does not work for you, you can always search the internet for other strategies.

**Comment by R3ntal-K4R** [comment]
try this: https://github.com/CuvtPi1L/ejs-tailwind-demo
